src/Streamly/Unicode/Stream.hs

```diff
@@ -69,13 +69,16 @@ module Streamly.Unicode.Stream
     -- * Construction (Decoding)
       decodeLatin1
```

Comment:
Since this is a new module in this release we should add "since 0.8.0" for all routines here. We forgot this when renaming this.